lib/workers/repository/process/index.ts

```diff
@@ -76,7 +76,7 @@ async function getBaseBranchConfig(
     baseBranchConfig.baseBranches = config.baseBranches;
   }
 
-  if (config.baseBranches!.length > 1) {
+  if (config.multipleBaseBranches) {
```

Comment:
Instead of adding a new field, we should IMO use a utility function here.

We are risking that we get a divergent state between these two in the future

Reply:
Tried it in the last commit